yal/main.py

```diff
@@ -233,6 +233,19 @@ async def process_message(self, message):
                 elif keyword == "i m not interested":
                     self.state_name = AssessmentApplication.NOT_INTERESTED_STATE
                 else:
+                    endline_max_limit = await rapidpro.get_rapidpro_global(
+                        "endline_max"
```

Comment:
Can we make this Global variable name a little bit more descriptive please? `Endline study max participant count` or something